src/pat/contentbrowser/src/App.svelte

```diff
@@ -23,7 +23,7 @@
     export let rootUrl = "";
     export let basePath = "";
     export let selectableTypes = [];
-    export let browseableTypes = ["Folder"];
+    export let browseableTypes = ["Folder", "LIF", "LRF"];
```

Comment:
@petschki is this list somehow configurable? Say you have a content-type that is folderish, or collective.folderishtypes installed, could we browse in such content-types?

Reply:
Sure, you can define this as pattern option with autoform hints like:
```
directives.widget("my_field", ContentBrowserFieldWidget, pattern_options={"browseableTypes": ["Folder", "LIF", "LRF", "MyType", ...]}
```

you can also define this as "global" pattern_options with the registry entry `plone.patternoptions` and add:

```JSON
{
    "contentbrowser": {
        "browseableTypes": [
            "Folder",
            "LIF",
            "LRF",
            "MyType"
        ]
    }
}
```

keep in mind, that the type list is not merged ... you will have to provide "Folder" and possible multilanguage types too otherwise you're stuck ... could be improved though ...

Reply:
I have done a test and I can browse any Folderish type in the contentbrowser, so I would say that my question is irrelevant :angel: